app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -538,7 +538,7 @@ def yum_names_for_job_template(action:, search:, versions: nil)
           if pkg_name_archs.empty?
```

Comment:
The line above this still searches the host's `installed_packages`, and then errors if an installed package is not found. Doesn't this need updating as well?